documentation/src/main/markdown/packagemgrs/sbt.md

```diff
@@ -34,6 +34,8 @@ In some cases, the dot files are not generated in "./target" and while those gra
 
 **NOTE: Older SBT projects that generate a resolution cache are still supported but are being deprecated. You must install the plugin for SBT to continue working uninterrupted.
 
+Evicted dependencies in Simple Build Tool(SBT) projects will not be included in the Bill of Materials(BoM) generated during the scan.
```

Comment:
Please add a note that this is with version 8.8.0 + of Detect.

Reply:
Done